pilot/pkg/networking/core/v1alpha3/listener_address.go

```diff
@@ -73,7 +73,7 @@ func getPassthroughBindIPs(ipMode model.IPMode) []string {
 
 	// it means that ipMode is empty if passthroughBindIPAddresses is empty
 	if len(passthroughBindIPAddresses) == 0 {
-		return []string{InboundPassthroughBindIpv4}
+		return []string{InboundPassthroughBindIpv6}
 	}
 
 	if !features.EnableDualStack && ipMode == model.Dual {
```

Comment:
This doesnot make sense, if ipMode = Dual, it will return ealier in L72

Reply:
It seems that function would not return at L72, it just an assignment statement.

Reply:
My mistake.  But it's better to check the most possible branch first

Reply:
okay, I understand that you mean this check should be the beginning of this function